modules/extensions-soap-support/src/test/java/org/mule/runtime/module/extension/soap/internal/loader/SoapExtensionDeclarationTestCase.java

```diff
@@ -80,8 +90,20 @@ public void assertSoapExtensionModel() {
     assertConnectionProvider(providers.get(2), CALCIO_ID + "-connection", CALCIO_DESC);
   }
 
+  private void assertErrorModels(Set<ErrorModel> errors) {
+    assertThat(errors, hasSize(13));
+    ImmutableList<String> errorNames = ImmutableList.<String>builder()
+        .addAll(of(SoapErrors.values()).map(Object::toString).collect(toList()))
+        .add("RETRY_EXHAUSTED")
```

Comment:
no way of getting these default ones from somewhere? this test will break each time we add a new default one